presto-main/src/test/java/com/facebook/presto/operator/aggregation/AbstractTestAggregationFunction.java

```diff
@@ -46,7 +46,7 @@
     protected Session session;
 
     @BeforeClass
-    public final void initTestAggregationFunction()
+    public void initTestAggregationFunction()
```

Comment:
Is this a necessary change?

Reply:
Yes, otherwise we cannot override it in the child class `TestMergeStatisticalDigestFunction`, which we need to do to have access to the `FeaturesConfig` to enable the t-digest flag (set to false by default).